docs/source/get-started/installation.rst

```diff
@@ -76,21 +76,29 @@ This command installs the bleeding edge ``main`` version of BentoML, which is us
 Editable install
 ----------------
 
-You may want an editable install to:
-
-- Stay up-to-date with the latest features and bug fixes;
-- Contribute to the BentoML project and test code changes.
-
-Clone the repository to your local folder and install BentoML with ``pip``:
+Install BentoML in editable mode is useful when you are testing your code changes in a local copy of the BentoML project. First, Clone the repository locally and install BentoML with ``pip install -e``:
 
 .. code-block:: bash
 
     git clone https://github.com/bentoml/bentoml.git
     cd bentoml
     pip install -e .
 
-This command installs BentoML in `editable mode <https://pip.pypa.io/en/stable/topics/local-project-installs/#editable-installs>`_, which allows you to install the project without copying any files. Python links this folder and adds it to Python library paths. This means that any changes to the folder can be tested immediately. For more information, see the `Developer Guide <https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md>`_.
+This command installs BentoML in `editable mode <https://pip.pypa.io/en/stable/topics/local-project-installs/#editable-installs>`_, which allows you to test any local code changes immediately by importing the ``bentoml`` library again or running a BentoML CLI command. For more information, see the `Developer Guide <https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md>`_.
+
+.. note::
+
+    In editable mode, Python links this folder and adds it to Python library paths. To keep using the library, you must not move or delete the ``bentoml`` folder after installing it in editable mode.
+
+Deploy With Custom Branch
```

Comment:
In fact, if you are using a local copy of bentoml, it will be packed into the bento and used as the runtime, no matter it is  pushed to GitHub or not.

Reply:
make sense, updated!